certified-connectors/YakChat/apiProperties.json

```diff
@@ -0,0 +1,51 @@
+{
+  "properties": {
+    "connectionParameters": {
+      "token": {
+        "type": "oauthSetting",
+        "oAuthSettings": {
+          "identityProvider": "aad",
+          "clientId": "314772fb-53a1-4908-9b17-1692796bb13f",

```

Comment:
@aMatthewsYakChat Please replace it with "[DUMMY]"

Reply:
This has been updated